lib/utils/log/slog_text_handler.go

```diff
@@ -284,18 +288,17 @@ func formatComponent(value slog.Value, padding int) string {
 }
 
 func (s *SlogTextHandler) clone() *SlogTextHandler {
-	// We can't use assignment because we can't copy the mutex.
```

Comment:
I removed this comment since it seems to be no longer be relevant. I assume that by "assignment" we meant something like this:

```go
	clone := *s
	return &clone
```

We no longer have a mutex to worry about since it's encapsulated within `out` which will be reused between clones. I didn't replace the body of this function with an assignment though because I noticed that we have explicit calls to `slices.Clip` for some fields.